sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/flows_v1/metadata.yaml

```diff
@@ -9,3 +9,19 @@ labels:
 scheduling:
   dag_name: bqetl_cjms_nonprod
   query_project: moz-fx-data-shared-prod
+  # delay aggregates by 2 hours, to ensure data is complete
+  date_partition_parameter: null
+  destination_table:
+    >-
+    flows_v1${{
+    (execution_date - macros.timedelta(hours=2)).strftime("%Y%m%d")
+    }}
+  parameters:
+    - >-
+      submission_date:DATE:{{
+      (execution_date - macros.timedelta(hours=2)).strftime("%F")
```

Comment:
IMO `%Y-%m-%d` would be more self-explanatory (e.g. the [Python docs](https://docs.python.org/3.8/library/datetime.html#strftime-and-strptime-format-codes) don't cover the `%F` format code).

Reply:
```suggestion
      (execution_date - macros.timedelta(hours=2)).strftime("%Y-%m-%d")
```